From 4261e7f29e5b37f9a55f636db0f095c2e3ad322c Mon Sep 17 00:00:00 2001
From: Ranjan Dutta <ranjan.dutta@intel.com>
Date: Tue, 6 Sep 2022 07:43:10 +0800
Subject: [PATCH 03/60] Revert "drm/meson: Fix refcount bugs in
 meson_vpu_has_available_connectors()"

This reverts commit 3aa710e96747c8b4e52ba12ffe09edcb2755897c.
---
 drivers/gpu/drm/meson/meson_drv.c | 5 +----
 1 file changed, 1 insertion(+), 4 deletions(-)

diff --git a/drivers/gpu/drm/meson/meson_drv.c b/drivers/gpu/drm/meson/meson_drv.c
index 2d022f3fb437e..728fea5094124 100644
--- a/drivers/gpu/drm/meson/meson_drv.c
+++ b/drivers/gpu/drm/meson/meson_drv.c
@@ -116,11 +116,8 @@ static bool meson_vpu_has_available_connectors(struct device *dev)
 	for_each_endpoint_of_node(dev->of_node, ep) {
 		/* If the endpoint node exists, consider it enabled */
 		remote = of_graph_get_remote_port(ep);
-		if (remote) {
-			of_node_put(remote);
-			of_node_put(ep);
+		if (remote)
 			return true;
-		}
 	}
 
 	return false;
-- 
2.25.1

